src/BuildScriptGenerator/Helpers/StaticSiteGeneratorHelper.cs

```diff
@@ -70,39 +81,30 @@ public static bool IsHugoApp(ISourceRepo sourceRepo)
                 jsonFilePaths.AddRange(configSourceRepo.EnumerateFiles("*.json", true));
             }
 
-            if (tomlFilePaths.Any())
+            foreach (var path in tomlFilePaths)
             {
-                foreach (var path in tomlFilePaths)
+                var tomlTable = ParserHelper.ParseTomlFile(sourceRepo, path);
+                if (tomlTable.Keys.Any(k => hugoVariables.Contains(k)))
```

Comment:
Are the keys case sensitive?

Reply:
Just verified they aren't case sensitive, so I had a comparer to ignore case sensitivity.